packages/visibility_detector/test/render_visibility_detector_test.dart

```diff
@@ -4,10 +4,11 @@
 // license that can be found in the LICENSE file or at
 // https://developers.google.com/open-source/licenses/bsd
 
+// ignore_for_file: invalid_use_of_protected_member
```

Comment:
:(

Reply:
These tests do some weird things that normally don't happen - if I don't explicitly call endRecording the painting context ends up in a bad state.

There's probably some other way to fix this but I got tired of trying to figure out what needed to change here.

Reply:
Maybe limit the ignores to the lines in question to avoid further unintentional violations of this in this file.